### ISSUE TITLE
Commit 6362b3f needs to update UI packages.

### ISSUE THREAD

**Issue by hiroyuki-sato** [treasure-data/digdag#755]
It seems that The commit 6362b3f0162b44ca73eb27b42eb3ae8ee2a28ef5 needs to update ui packages.

- [x] 1. `./gradlew cli` failed #753
- [ ] 2. After fix No.1, build succeed but http://localhost:65432 show blank page. 

It seems that No.1 fix with the following patch.
No.2 doesn't fix yet. 

I'm not familiar UI developing.
@dzannotti, Would you mind checking this issue?

Best regards.

```diff
diff --git a/digdag-ui/package.json b/digdag-ui/package.json
index c1b190ecb..2af09d411 100644
--- a/digdag-ui/package.json
+++ b/digdag-ui/package.json
@@ -69,14 +69,14 @@
     "less": "2.7.2",
     "less-loader": "2.2.3",
     "node-fs-extra": "^0.8.1",
-    "node-sass": "^3.10.0",
+    "node-sass": "^4.8.3",
     "resolve-url-loader": "2.0.2",
     "rimraf": "^2.5.4",
     "sass-loader": "^6.0.3",
     "standard": "^9.0.2",
     "style-loader": "0.16.1",
     "url-loader": "0.5.8",
-    "webpack": "1.13.1",
+    "webpack": "4.5.0",
     "webpack-dev-server": "1.14.1"
   },
   "standard": {
```

**Comment by hiroyuki-sato** [comment]
#760 fix this issue Thank you @Jerry-Hong and [@i_szyn](https://twitter.com/i_szyn/status/984022265288421376)

Build succeed after apply #760. 
* npm: 4.1.2 (npm --version)
* node: 7.7.3 (node --version)

**Comment by dzannotti** [comment]
The `node-sass` upgrade is necessary, i don't think the webpack one is? (and might need config update) cc @muga

**Comment by hiroyuki-sato** [comment]
Hello, @dzannotti 

Thank you for the comment.

It seems that the root cause was [resolve](https://gist.github.com/szyn/0fc71990eaa59f5ec060f9c5f3f2d846#file-2018-4-11-npm-run-dev-log-L884-L892) issue. #760.
What do you think upgrade [bootstrap-loader](https://github.com/treasure-data/digdag/pull/760#issuecomment-380415054)?

**Comment by hiroyuki-sato** [comment]
Update

Resolve 1.7.1 has been released. this version fixed resolve 1.7.0 issue. 
So we don't need change package anymore. (about build digdag)

**Comment by dzannotti** [comment]
I'm glad it was all sorted in the latest release. Great to know!
